src/main/java/com/uploadcare/api/Client.java

```diff
@@ -252,26 +571,126 @@ public void close() {
     }
 
     /**
+     * @deprecated Use {@link #copyFileLocalStorage(String, Boolean, Boolean)}
+     * or {@link #copyFileRemoteStorage(String, String, Boolean, String)} instead.
+     *
      * @param fileId  Resource UUID
      * @param storage Target storage name
      * @return An object containing the results of the copy request
      */
+    @Deprecated
     public CopyFileData copyFile(String fileId, String storage) {
+        return copyFileRemoteStorage(fileId, storage, true, null);
```

Comment:
but what about local copy?
```suggestion
        if (storage != null && !storage.isEmpty()) {
            return copyFileRemoteStorage(fileId, storage, true, null);
        } else {
            return copyFileLocalStorage(fileId, true, true);
        }
```